doc/nrf/protocols/matter/overview/group_communication.rst

```diff
@@ -0,0 +1,259 @@
+
+      #. Send a command to toggle on/off on the application endpoint (endpoint 1 includes the onoff cluster)::
+
+         ./chip-tool onoff toggle 0xffffffffffffdcba 1
+
+      **LED2** (nRF53 and nRF52 Series devices) or **LED1** (nRF54 Series devices) on the Light Bulb should now change the state.
+
+Group Communication triggered from the accessory (Light Switch)
+===============================================================
+
+The initial settings for this use case are the following:
+
+.. code-block:: none
+
+   Light Bulb nodeid = 2
+   Light Switch nodeid = 1
+   Test group keyset
+        groupKeySetID=258
+        epochKey0=a0a1a2a3a4a5a6a7a8a9aaabacad7531
+        epochStartTime0=1110000
+        epochKey1=b0b1b2b3b4b5b6b7b8b9babbbcbd7531
+        epochStartTime1=1110001
+        epochKey2=c0c1c2c3c4c5c6c7c8c9cacbcccd7531
+        epochStartTime2=1110002
+   Test group:
+        groupId=30001
+        name=Test_Group_30001
+
+.. tabs::
+
+   .. group-tab:: Nodes
+
+      You need the following to run through this use case:
+
+      * The `chip-tool`_
+      * The :ref:`matter_light_bulb_sample` sample
+      * The :ref:`matter_light_switch_sample` sample
+
+   .. group-tab:: Precondition
+
+      Commission the Light Bulb and Light Switch to the same fabric as the chip-tool.
+
+   .. group-tab:: Steps
+
+      Complete the following steps to start Group Communication:
+
+      1. Set the group settings on the on/off client (Light Switch)::
+
+         ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"258","groupKeySecurityPolicy":0,"epochKey0":"a0a1a2a3a4a5a6a7a8a9aaabacad7531","epochStartTime0":"1110000","epochKey1":"b0b1b2b3b4b5b6b7b8b9babbbcbd7531","epochStartTime1":"1110001","epochKey2":"c0c1c2c3c4c5c6c7c8c9cacbcccd7531","epochStartTime2":"1110002"}' 1 0
+
+      #. Set the keyset to the group map on the on/off client (Light Switch)::
+
+         ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"30001","groupKeySetID":"258","fabricIndex":"1"}]' 1 0
+
+      #. Set the same group settings on the on/off server (Light Bulb)::
+
+         ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"258","groupKeySecurityPolicy":0,"epochKey0":"a0a1a2a3a4a5a6a7a8a9aaabacad7531","epochStartTime0":"1110000","epochKey1":"b0b1b2b3b4b5b6b7b8b9babbbcbd7531","epochStartTime1":"1110001","epochKey2":"c0c1c2c3c4c5c6c7c8c9cacbcccd7531","epochStartTime2":"1110002"}' 2 0
+
+      #. Set the keyset to the group map on the on/off server (Light Bulb)::
+
+         ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"30001","groupKeySetID":"258","fabricIndex":"1"}]' 2 0
+
+      #. Add the group to the application endpoint (endpoint 1 includes the on/off cluster)::
```

Comment:
Does it apply to light bulb, light switch or both? The other steps point to which device it applies directly.

Reply:
you are right, it should be more precisely mentioned that it should be set only on light bullb (which support group cluster on application endpoint)

Reply:
Please, suggest how to put this right.

Reply:
We should change the step to `Add the group to the Light Bulb Application endpoint (endpoint 1 includes the on/off cluster)`